src/cogent3/core/new_alignment.py

```diff
@@ -650,16 +671,36 @@ def to_dict(self, as_array: bool = False) -> dict[str, Union[str, numpy.ndarray]
         get = self.seqs.get_seq_array if as_array else self.seqs.get_seq_str
         return {name: get(seqid=name) for name in self.names}
 
-    def to_json(self):
-        # todo
-        # refactor: design - waiting for alphabet serialisation
-        ...
+    def to_rich_dict(self):
+        """returns a serialisable string"""
+        # refactor: design
+        # consider whether we want to include a from_rich_dict method
+        data = {}
+        moltype = self.moltype.label
+        info = {} if self.info is None else self.info
 
-    def to_rich_dict(self) -> dict:
-        # todo
-        # refactor: design - waiting for alphabet serialisation
-        # add SeqsData.to_dict method which returns a dict [str: str]
-        ...
+        if info.get("Refs", None) is not None and "Refs" in info:
+            info.pop("Refs")
+
+        info = info or None
+
+        data = dict(
+            seqs=self.seqs.to_rich_dict(),
+            moltype=moltype,
+            names=self.names,
+            info=info,
+            type=get_object_provenance(self),
+            version=__version__,
+        )
+
+        if hasattr(self, "annotation_db") and self.annotation_db:
+            data["annotation_db"] = self.annotation_db.to_rich_dict()
+
+        return data
+
+    def to_json(self):
```

Comment:
you need to write a deserialise function for this SequenceCollection class and register it, next PR